NEWS.md

```diff
@@ -1,6 +1,8 @@
 # simstudy (development version)
 
 ## New features
+* Updated genFormula with functionality to generate formulas with double dot variables inside.
```

Comment:
I think there was one more thing that I was supposed to put on here but it's slipped my mind